main.go

```diff
@@ -40,7 +42,12 @@ func main() {
 		log.Fatalln(err)
 	}
 
-	// retrieve raw from graphql
+	// processed separately from env map above because optional w/ no default val
+	masterBundleSha := os.Getenv("MASTER_BUNDLE_SHA256")
```

Comment:
You don't have to adhere to the env var naming of our a-i PR check pipeline. Pick what makes sense, e.g. also a cli flag over an env var if that suits the UX of this tool better. 

You can always adapt in your hack runner func.